apk-generator/v2/app/generator/generator.py

```diff
@@ -310,6 +313,7 @@ def update_status(self, state, exception=None, message=None, skip_log=False):
                 self.task_handle.update_state(
                     state=state, meta=meta
                 )
+        self.handle_message(message)
```

Comment:
`message` is always `None` until the app generation is complete. You probably want to use `state` here. See comments in #1821